cypress/utils/collectionsUtils.js

```diff
@@ -1,15 +1,10 @@
-import Collections from "../apiRequests/Collections";
+import { deleteCollection, getCollections } from "../apiRequests/Collections";
 
-class CollectionsUtils {
-  collectionsRequest = new Collections();
-
-  deleteProperCollections(startsWith) {
-    cy.deleteProperElements(
-      this.collectionsRequest.deleteCollection,
-      this.collectionsRequest.getCollections,
-      startsWith,
-      "collection"
-    );
-  }
+export function deleteProperCollections(startsWith) {
```

Comment:
```suggestion
export function deleteCollectionsStartingWith(startsWith) {
```

Or, if we always create objects with `cypress` prefix, maybe helper which will be called `deleteTestCollections` would be better?

Reply:
I rename this functions to deleteCollections/ProductsStartsWith. If in the future we would like to run tests in parallel, there shouldn't be always cypress prefix.